api-service/crowdflower/crowdflower.js

```diff
@@ -0,0 +1,36 @@
+<iframe
+  src="https://crowdsource-demo-dot-wikidetox.appspot.com/?embedded=true#{{client_job_key}}/{{question_id}}"
+  width="400"
+  height="425"
+  frameborder="0"
+  scrolling="no"
+  marginheight="0"
+  marginwidth="0"
+  >
+</iframe>
+
+<cml:hidden
+  default="test"
+  question-label="Sample text field:"
+  validates="required"
+  value="test"
+  name="question_comments">
+</cml:hidden>
+
+<script>
+require(['jquery-noconflict'], function($) {
+    //Ensure MooTools is where it must be
+    Window.implement('$', function(el, nc){
```

Comment:
Small nit: I prefer full variable names instead of 2 letter ones like el, nc. I know this is just boilerplate, but it helps with readability.

Reply:
This is copy and pasted from the CrowdFlower documentation (https://success.crowdflower.com/hc/en-us/articles/205517285-Javascript-How-to-Add-jQuery) so I'd rather keep it identical. But I hear you on the two letter variable names!